orbit-2-core/README.md

```diff
@@ -93,6 +94,40 @@ Orbit MVI system. It retains the state, allows you to listen to side effects and
 state updates and allows you to modify the state through the `orbit` function
 which executes Orbit operators of your desired business logic.
 
+### Subscribing to the container
+
+[Container](src/main/java/com/babylon/orbit2/Container.kt) exposes flows
+that emit updates to the container state and side effects.
+
+- State emissions are conflated
+- Side effects are cached by default if no observers are listening. This
+  can be changed via
+  [Container Settings](src/main/java/com/babylon/orbit2/Container.kt#Settings)
+
+``` kotlin
+data class ExampleState(val seen: List<String> = emptyList())
+
+sealed class ExampleSideEffect {
+   data class Toast(val text: String)
+}
+
+fun main() {
+    // create a container
+    val container = container<ExampleState, ExampleSideEffect>(ExampleState())
+
+    // subscribe to updates
+    CoroutineScope(Dispatchers.Main).launch {
```

Comment:
worth adding a note for Android? use `lifecycleScope.launch(Dispatchers.Main)` instead??